planex/cmd/mock.py

```diff
@@ -50,6 +51,22 @@ def parse_args_or_exit(argv=None):
     return parser.parse_args(argv)
 
 
+def pty_check_call(cmd):
+    # python2.7 doesn't return the exitcode here:
+    pty.spawn(cmd)
+    # get exit status of first child
+    (pid, status) = os.waitpid(-1, 0)
+    if status != 0:
+        if os.WIFEXITED(status):
```

Comment:
Thanks, force pushed the PR branch. (For the future/in general do you prefer force pushes for updating PRs under review, or extra commits with `git commit --fixup`/`git commit --squash`?)